client/js/components/procedure/StatementSegmentsList/StatementSegment.vue

```diff
@@ -259,7 +260,9 @@
           }"
           @click.prevent="showSegmentVersionHistory"
           data-cy="segmentVersionHistory">
-          <dp-icon icon="history" />
+          <dp-icon
+            class="inline-block"
```

Comment:
should this be added in dpIcon? Or are there cases, whee its not usefull to set it to inline-block?

Reply:
this was my first thought as well, but then, i realised that the "new standard" way of doing micro ui would rather involve `inline-flex` which did not work for this special case. Iguess more research is needed to set a reasonable default for the `display` property in `DpIcon`.